src/model/resources/thermal/thermal.jl

```diff
@@ -34,9 +36,18 @@ function thermal!(EP::Model, inputs::Dict, setup::Dict)
             for y in THERM_ALL))
         add_similar_to_expression!(EP[:eCapResMarBalance], eCapResMarBalanceThermal)
 
-        MAINT = ids_with_maintenance(gen)
         if !isempty(intersect(MAINT, THERM_COMMIT))
             thermal_maintenance_capacity_reserve_margin_adjustment!(EP, inputs)
         end
+
+        if !isempty(intersect(FUSION, THERM_COMMIT))
```

Comment:
Is it possible for FUSION plant indices not to be in THERM_COMMIT?

Reply:
Not right now, but we could consider a THERM_NOCOMMIT fusion later.